### PR TITLE
Add  unit tests to project

### PR DESCRIPTION
**Pull request by mojoLyon**

### CODE REVIEW

**Comment by cedvan** [comment]
:+1:

File: tests/Units/Command/HttpProcessCommand.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            ->given(
+                $loopMock       = $this->getLoopMock(),
+                $socketMock     = $this->getSocketMock(),
+                $httpServerMock = $this->getHttpServerMock(),
+                $containerMock  = $this->getContainer($loopMock, $socketMock, $httpServerMock),
+                $inputMock      = $this->getInputMock($port = 8000, $memoryMax = 200, $checkInterval = 60),
+                $outputMock     = $this->getOutputMock(),
+                $testedCommand    = new TestedCommand(),
+                $testedCommand->setContainer($containerMock),
+                $this->function->memory_get_peak_usage = 412467200
```
Comment (omansour) [inline tests/Units/Command/HttpProcessCommand.php:51]: merci Atoum :) :yellow_heart:

**Comment by gbouyge** [comment]
:+1:

**Comment by fabdsp** [comment]
:+1:

**Pull request merged**